docs/0.1/creating_organizations.md

```diff
@@ -0,0 +1,236 @@
+
+   This example uses the base name `alpha-agent` to indicate that you will be
+   acting as your organization's agent to add products and other Grid items
+   from the alpha node.
+
+   ```
+   root@gridd-alpha:/# grid keygen alpha-agent
+   Writing /root/.grid/keys/alpha-agent.priv
+   Writing /root/.grid/keys/alpha-agent.pub
+   ```
+
+### Set Environment Variables
+
+Set the following Grid environment variables to simplify entering the
+`grid` commands in this procedure.
+
+{:start="3"}
+
+3. Set `GRID_DAEMON_KEY` to the base name of the agent's public/private key
+   files (such as `alpha-agent`). This environment variable replaces the
+   `-k` option on the `grid` command line.
+
+   Tip: If you're using this example docker-compose environment, this
+   variable is already defined for the `gridd-alpha` and `gridd-beta` containers.
+
+   ```
+   root@gridd-alpha:/# export GRID_DAEMON_KEY="alpha-agent"
+   ```
+
+   **Note**: Although this variable has "daemon" in the name, it should
+   reference the user key files in `$HOME/.grid/keys`, not the Grid daemon's key
+   files in `/etc/grid/keys`.
+
+1. Set `GRID_DAEMON_ENDPOINT` to the endpoint for the node's `gridd` container
+   (such as `https://localhost:8080`). This environment variable replaces the
+   `--url` option on the `grid` command line.
+
+   Tip: If you're using this example docker-compose environment, this variable
+   is already defined for the `gridd-alpha` and `gridd-beta` containers.
+
+   ```
+   root@gridd-alpha:/# export GRID_DAEMON_ENDPOINT="https://localhost:8080"
+   ```
+
+1. For Grid on Splinter: Set `GRID_SERVICE_ID` to the fully qualified service ID
+   for the scabbard service on this circuit (such as `01234-ABCDE::gsAA`).
+   This environment variable replaces the `--service_id` option on the `grid`
+   command line.
+
+   ```
+   root@gridd-alpha:/# export GRID_SERVICE_ID=01234-ABCDE::gsAA
+   ```
+
+### Create an Organization
+
+{:start="6"}
+
+6. Create a new organization by specifying a unique organization ID, the
+   organization's name and street address, and optional metadata (as key-value
+   strings).
+
+   This example uses the ID `314156`, the name `myorg`, an imaginary street
+   address, and GS1-specific metadata to note that the ID is a GS1 company
+   prefix.
+
+   ```
+   root@gridd-alpha:/# grid organization create \
+   314156 myorg '123 main street' \
+   --metadata gs1_company_prefixes=314156
+   ```
+
+   This command creates and submits a transaction to create a new Pike
+   organization with the data you supplied. The transaction is signed with the
+   agent's private key, as derived from the base name specified by
+   `GRID_DAEMON_KEY` (or with the `-k` option on the command line).
+
+   This command also creates a new Pike agent, using the agent's public key (as
+   derived from the same base name) as the agent ID. This command automatically
+   assigns the admin role to the new agent, but does not make the agent active
```

Comment:
This is incorrect, the agent is active

Reply:
@agunde406 - If the agent is active, why do you have to specify `--active` in the next command?

Reply:
It is a required flag. Either `--active` or `--inactive` must be set on that command. This is done so an agent is never accidentally set to active/inactive when it is not the intended result.